actinia_module_plugin/api/modules/grass.py

```diff
@@ -91,3 +94,21 @@ def get(self, grassmodule):
                 message='Error looking for module "' + grassmodule + '".'
             )))
             return make_response(res, 404)
+
+    def post(self, grassmodule):
+        """Install an official GRASS GIS Addon.
+        """
+
+        response = installGrassAddon(self, grassmodule)
+        addGrassAddonToModuleListRedis(self, grassmodule)
```

Comment:
A bit late because already merged: add the new addon to the user's module list in redis only if was successfully installed.

Reply:
added in #28